### PR TITLE
Frontend redesign by Axel

### PR DESCRIPTION
**Pull request by hyeomans**
Screen Shot 2021-05-03 at 10 25 56
Screen Shot 2021-05-03 at 10 25 40
Screen Shot 2021-05-03 at 10 25 23
Screen Shot 2021-05-03 at 10 10 25

### CODE REVIEW

**Comment by jmsalcido** [comment]
:shipit:

File: src/components/Nav.js [hunk 1]
```diff
@@ -15,7 +17,9 @@ export default function Nav() {
             d="M9 20l-5.447-2.724A1 1 0 013 16.382V5.618a1 1 0 011.447-.894L9 7m0 13l6-3m-6 3V7m6 10l4.553 2.276A1 1 0 0021 18.382V7.618a1 1 0 00-.553-.894L15 4m0 13V4m0 0L9 7"
           />
         </svg>
-        <span className="font-semibold text-xl ml-1">¿Quién me representa?</span>
+        <Link href="/">
+          <a className="font-semibold text-xl ml-1">¿Quién me representa?</a>
+        </Link>
```
Comment (keogh) [inline src/components/Nav.js:22]: an `a` inside a `Link`? is that really necessary?

File: src/pages/index.js [hunk 2]
```diff
@@ -145,44 +162,34 @@ export default function Home() {
...
+    <div className="h-screen w-screen bg-gray-100 flex flex-col justify-between">
+      <Nav />
+      <div className="relative w-full h-full">
+        <Map
+          className="absolute inset-0 z-0"
+          center={[23.681406310669356, -103.31439396326121]}
+          zoom={4}
+          scrollWheelZoom={true}>
+          {({ TileLayer, Marker, useMapEvents }) => (
+            <>
```
Comment (keogh) [inline src/pages/index.js:174]: No need for fragment here

**Review by keogh — COMMENTED** [review]
👍 Solo un par de non-blocking comments

**Pull request merged**